pkg/config/config.go

```diff
@@ -143,8 +144,15 @@ func checkFilePerms(files ...string) error {
 			continue
 		}
 
-		if fInfo.Mode() != 0600 {
-			return errors.E(op, f+" should have 0600 as permission")
+		if runtime.GOOS == "windows" {
+			if fInfo.Mode() != 0400 {
```

Comment:
I think this should be 0600 @michalpristas, this lines up with unix perms. What do you think?